platform-controller/src/main/java/org/hobbit/controller/PlatformController.java

```diff
@@ -327,52 +302,24 @@ public void run() {
     }
 
     /**
-     * This method closes the command queue and its handling and reopens it on the
-     * RabbitMQ broker which will be used for experiments.
+     * This method sets the RabbitMQ connector for the command queue.
+     */
+    public void setExpRabbitMQConnector(RabbitMQConnector rabbitMQConnector) {
+        LOGGER.info("Setting experiment's RabbitMQ connector for the command queue: {}", rabbitMQConnector);
+        assert this.rabbitMQConnector == null : "RabbitMQ connector should be null";
```

Comment:
Using `assert` at runtime is not a good idea. Please throw an exception or try to handle the case in a different way.